docs/scenarios_authoring/RRDataSourcesAndSinks.md

```diff
@@ -0,0 +1,45 @@
+---
+sidebar_position: 4
+---
+
+# Request-Response sources and sinks
+
+## Http request as input and output to Nussknacker scenarios
+
+### Prerequisites
+
+To fully understand how Nussknacker works in Request-Response mode, it's best to read the following first:
```

Comment:
Request-response 
:-(

Reply:
Nooo, I don't think this is a prerequisite :) 
We don't expose any headers or the likes, we even hide HTTP method

Reply:
maybe https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview site gives better overview wdyt ?